src/test/js/log-karaoke/stages.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
             })
         ;
         // test whether the expand works
-        nodeDetail.clickFirstResultItem(true);
+        nodeDetail.clickFirstResultItem();
```

Comment:
I agree ... making it explicit is def a good thing. Those 4 extra characters "true" are not noise, they are relevant information.